lnwire/writer_test.go

```diff
@@ -0,0 +1,606 @@
+package lnwire
+
+import (
+	"bytes"
+	"encoding/base32"
+	"image/color"
+	"math"
+	"net"
+	"testing"
+
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/btcsuite/btcd/wire"
+	"github.com/btcsuite/btcutil"
+	"github.com/lightningnetwork/lnd/tor"
+	"github.com/stretchr/testify/require"
+)
+
+func TestWriteBytes(t *testing.T) {
+	buf := new(bytes.Buffer)
+	data := []byte{1, 2, 3}
+
+	err := WriteBytes(buf, data)
+
+	require.NoError(t, err)
```

Comment:
error strings with require usage?

Reply:
I intentionally leave it there in this test as I think the returned error message is informative enough,
```
--- FAIL: TestWriteBytes (0.00s)
    writer_test.go:26: 
        	Error Trace:	writer_test.go:26
        	Error:      	Received unexpected error:
        	            	wrong data
        	Test:       	TestWriteBytes
FAIL
exit status 1

```